hyperframe/flags.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+"""
+hyperframe/flags
+~~~~~~~~~~~~~~~~
+
+Defines basic Flag and Flags data structures.
+"""
+import collections
+try:
+    from collections.abc import MutableSet
+except ImportError:  # pragma: nocover
+    from collections import MutableSet
```

Comment:
This conditional import isn't necessary, I think: `collections` always exposes `collections.abc`'s members at the top level. I think I'll fix this myself though.